### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by ochococo**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [x] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with [GitFlow](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Grand Central Board for the Apple TV
 ====================================
 
-![image](./README/screenshot.png)
+![image](./README/appletv941.png)
```
Comment (michallaskowski) [inline README.md]: I think having a screenshot is more attractive - even more since we have more widgets.

File: README.md [hunk 2]
```diff
@@ -19,13 +18,70 @@ The board is a lightweight piece of code. The TV screen is to be used in landsca
...
+let configFileName = NSBundle.localConfigurationFileName
+let configurationFetching = LocalConfigurationLoader(configFileName: configFileName,
+                                                     availableBuilders: availableBuilders)
+
+let boardController = GrandCentralBoardController(scheduler: self.scheduler, stack: self.autoStack)
+
+let configurationRefresher = ConfigurationRefresher(interval: configRefreshInterval,
+                                                    configuree: boardController,
+                                                    fetcher: configurationFetching)
+```
```
Comment (michallaskowski) [inline README.md]: Could we add an instruction how to display the view of GCB? Like we have in `MainViewController`?

File: README.md (hunk 1 above)
Comment (ochococo) [inline README.md]: The widgets will leave this repository. 

But I agree if we have new and more attractive screenshot I would love to add here. The one Adam sent had an error displayed.

File: README.md (hunk 2 above)
Comment (ochococo) [inline README.md]: Right. Will fix it.

**Comment by michallaskowski** [comment]
Looks good. If you want to go with the icon, you can merge it, or we can create a better screenshot.

**Pull request merged**